File: example/android/app/src/main/kotlin/com/flutter/twilio/voice_example/TwilioVoicePluginRegistrant.java
package com.flutter.twilio.voice_example;

import com.flutter.twilio.voice.TwilioVoice;

import io.flutter.plugin.common.PluginRegistry;

public final class TwilioVoicePluginRegistrant{
    public static void registerWith(PluginRegistry registry)
    {
        if (alreadyRegisteredWith(registry))
        {
            return;
        }
        TwilioVoice.registerWith(registry.registrarFor("com.flutter.twilio.voice"));
    }

    private static boolean alreadyRegisteredWith(PluginRegistry registry)
    {
        final String key = TwilioVoicePluginRegistrant.class.getCanonicalName();
        if (registry.hasPlugin(key))
        {
            return true;
        }
        registry.registrarFor(key);
        return false;
    }
}

File: example/android/app/src/main/kotlin/app/krispcall/TwilioVoicePluginRegistrant.kt
package app.krispcall

import com.flutter.twilio.voice.TwilioVoice.Companion.registerWith
import io.flutter.plugin.common.PluginRegistry

object TwilioVoicePluginRegistrant {
    fun registerWith(registry: PluginRegistry) {
        if (alreadyRegisteredWith(registry)) {
            return
        }
        registerWith(registry.registrarFor("com.flutter.twilio.voice"))
    }

    private fun alreadyRegisteredWith(registry: PluginRegistry): Boolean {
        val key = TwilioVoicePluginRegistrant::class.java.canonicalName
        if (registry.hasPlugin(key)) {
            return true
        }
        registry.registrarFor(key)
        return false
    }
}